viostor/virtio_pci.c

```diff
@@ -197,12 +197,12 @@ static u16 vdev_get_msix_vector(void *context, int queue)
             } else {
                 vector = queue + 1;
             }
-        } else {
-            vector = VIRTIO_MSI_NO_VECTOR;
         }
     } else {
         /* on-device-config-change interrupt */
-        vector = VIRTIO_MSI_NO_VECTOR;
+        if (!adaptExt->msix_one_vector) {
```

Comment:
please seem my comment at https://bugzilla.redhat.com/show_bug.cgi?id=1442322#c13
However, I can be completely wrong, and if you think that you know how to deliver two different messages using one vector, please share your thoughts.